src/server/tests-controller.js

```diff
@@ -1,31 +1,44 @@
 import PageRenderer from './page-renderer.js';
 import { PAGE_NAME } from '../domain/page-name.js';
-import { STORAGE_LOCATION_TYPE } from '../domain/constants.js';
+import RoomService from '../services/room-service.js';
+import { ROOM_ACCESS_LEVEL, STORAGE_LOCATION_TYPE } from '../domain/constants.js';
 import permissions, { hasUserPermission } from '../domain/permissions.js';
 
 class TestsController {
-  static get inject() { return [PageRenderer]; }
+  static get inject() { return [PageRenderer, RoomService]; }
 
-  constructor(pageRenderer) {
+  constructor(pageRenderer, roomService) {
     this.pageRenderer = pageRenderer;
+    this.roomService = roomService;
   }
 
-  handleGetTestsPage(req, res) {
+  async handleGetTestsPage(req, res) {
     const initialState = {};
 
+    const locations = [
+      {
+        type: STORAGE_LOCATION_TYPE.public,
+        rootPath: 'media',
+        initialPath: 'media',
+        uploadPath: 'media',
+        isDeletionEnabled: hasUserPermission(req.user, permissions.DELETE_ANY_STORAGE_FILE)
+      }
+    ];
+
+    const rooms = await this.roomService.getRoomsOwnedByUser(req.user?._id || 'hihi');
```

Comment:
These are the funny rooms of the funny user, unless a non funny user is provided 😂

Reply:
It's always good to fall back to the funniness we cherish so much :smile: